factory/config.go

```diff
@@ -263,6 +265,19 @@ func (c *Configuration) parseRocConfig(rsp *protos.NetworkSliceResponse) error {
 
 	c.EnterpriseList = make(map[string]string)
 
+	//should be updated to be received from webui.
+	//currently adding port info in webui causes crash.
+	pfcpPortStr := os.Getenv("PFCP_PORT_UPF")
```

Comment:
Should we do better solution if time permits.

Reply:
added fix to parse port from upf name.